test/getattr_test.py

```diff
@@ -29,12 +31,17 @@ def z(self):
         def make_foo():
             return Foo()
 
+        @classproperty
```

Comment:
Should we test this with an async method too, to check that it gets wrapped into method with .aio etc.?